packages/share/ui/src/lib/blog/blog-card/blog-card.tsx

```diff
@@ -0,0 +1,17 @@
+type BlogCardProps = { title: string; subtitle: string; imgUrl: string };
+export function BlogCard({ title, subtitle, imgUrl }: BlogCardProps) {
+  return (
+    <div className="relative aspect-[4/3] ">
+      <div
+        className="absolute w-full h-full -z-10"
+        style={{
+          background: `url(${imgUrl})`,
```

Comment:
don't use image as background, please.
since we haven't set up Nextjs Image yet, we can take image as children for now.